### DISCUSSION TITLE
change "gas" to natural gas so people don't think it's gasoline

### DISCUSSION THREAD

**Discussion by ecadvocate** [electricitymaps/electricitymaps-contrib#2663] [General]
change "gas" to natural gas so people don't think it's gasoline (yes, they do)

**Comment by Kongkille** [comment]
@tth4vb what do you think?

**Comment by tth4vb** [comment]
makes sense to me!

**Comment by simonhac** [comment]
how about "fossil gas" since other forms of gas (biogas, hydrogen, etc.) will become more prevalent over the century.

**Comment by gerrykou** [comment]
Natural gas makes more sense, agree.

**Reply by madsnedergaard** [reply]
Thanks for commenting - we have decided to go with "gas" only, as "natural gas" is as "natural" as coal.
See the PR here: https://github.com/electricitymaps/electricitymaps-contrib/pull/3903

<img width="646" alt="159748422-f4a37f49-2b0b-4efe-9f1f-8bd8a36a0b10" src="https://user-images.githubusercontent.com/3296643/198573581-79c0c0b5-16e2-4beb-84dc-c7587d6ac069.png">
